build.java

```diff
@@ -771,6 +799,7 @@ static void removeDependencies(Tasks.FileReplace.Effects effects, Function<Path,
         final Path suitePy = Path.of("substratevm", "mx.substratevm", "suite.py");
         final Path path = mandrelRepo.apply(suitePy);
         final List<String> dependencies = Arrays.asList(
+            "jdk.scripting.nashorn@11..14",
```

Comment:
Please add a comment like the following:

```suggestion
            "jdk.scripting.nashorn@11..14",  // work around https://github.com/graalvm/mandrel/issues/280
```

Reply:
@zakkak Addressed, squashed.